WordPress/gutenberg
# Changed retry icon color

**Fixes** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2429

## Related PRs:
gutenberg-mobile TBD
WordPress-iOS TBD
WordPress-Android TBD

## Description
This fix changes the colour of the retry icon in case of upload failure to match the colour of the text. This also aligns the design with other components (e.g. Image)

## How has this been tested?
* Add a Media-Text block
* Press the add image or video button
* Select a video and trigger an upload error (e.g. disconnect from the network)
* Confirm that the colour of the retry icon is white

## Screenshots

|Media text retry colour icon that matches the text|
|---|
|![Simulator Screen Shot - iPhone 11 - 2020-06-29 at 14 21 36](https://user-images.githubusercontent.com/304044/85999714-3c7b1400-ba15-11ea-93d2-3ca89a2256e9.png)|

## Types of changes
Bug fix

## Checklist:
